NETProvider/src/FirebirdSql.Data.UnitTests/TestsBase.cs

```diff
@@ -457,7 +453,17 @@ public static string BuildServicesConnectionString(bool includeDatabase)
 			return cs.ToString();
 		}
 
-		public static FbConnectionStringBuilder BuildConnectionStringBuilder()
+		public static IEnumerable<FbConnectionStringBuilder> BuildConnectionStringBuilder()
```

Comment:
Why is here connection string for all `FbServerType`s? The `TestFixture` attributes should handle "all" the cases, isn't it?